src/text_manipulation.py

```diff
@@ -0,0 +1,110 @@
+import random
+import spacy
+import utils
+
+class Manipulator:
+
+    def __init__(self):
+        self.nlp = spacy.load('en_core_web_sm')
+
+    def reorder_paragraphs(self, paragraph_list, random_seed=3142):
+
+        random.seed(random_seed)
+
+        paragraph_nested_list = []
+
+        for this_paragraph in paragraph_list:
+            paragraph_nested_list.append([this_paragraph])
+
+        random.shuffle(paragraph_nested_list)
+
+        return paragraph_nested_list
+
+    def print_nested_list(self, input_list):
+
+        for this_item in input_list:
+            print(this_item[0])
+            print(" ")
+
+    def print_list(self, input_list):
+
+        for this_item in input_list:
+            print(this_item)
+            print(" ")
+
+    #def swap_two_sentences(self, sentences_list):
+
+     #   idx = range(len(sentences_list))
+     #  i1, i2 = random.sample(idx, 2)
+     # sentences_list[i1], sentences_list[i2] = sentences_list[i2], sentences_list[i1]
+
+     #return sentences_list
+
+    def reorder_sentences(self, sentences_list):
+
+        import random
+
+        paragraph_list = list(sentences_list)
```

Comment:
Isn't sentences list already a list? If so you don't need to use the list() function, if not you update the variable name

Reply:
Yes it is.... i have deleted it out and changes all paragraph_list to sentences_list